zeebe/gateway-protocol/src/main/proto/v2/expression.yaml

```diff
@@ -0,0 +1,69 @@
+## Endpoint definitions
+
+paths:
+  /expression/evaluation:
+    post:
+      operationId: evaluateExpression
+      x-eventually-consistent: true
+      tags:
+        - Expression
+      summary: Evaluate a FEEL expression
+      description: Evaluates a FEEL expression with optional context variables and scope. Requires EXPRESSION:EVALUATE permission.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ExpressionEvaluationRequest'
+      responses:
+        '200':
+          description: Expression evaluated successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExpressionEvaluationResult'
+        "400":
+          $ref: 'common-responses.yaml#/components/responses/InvalidData'
+        "401":
+          $ref: 'common-responses.yaml#/components/responses/Unauthorized'
+        "403":
+          $ref: 'common-responses.yaml#/components/responses/Forbidden'
+        "500":
+          $ref: 'common-responses.yaml#/components/responses/InternalServerError'
+
+components:
+  schemas:
+    ExpressionEvaluationRequest:
```

Comment:
❓ Shouldn't this also contain a "context" I can provide for the expression evaluation?

Reply:
not in the scope of this EPIC, it's part of a followup [epic](https://docs.google.com/document/d/1ReAkpc31Y9nzfpOTct0wLpM2Ks4Du7T-snul4q6x-gQ/edit?tab=t.0#heading=h.hogjjld64a8s)